src/styles/product-card.scss

```diff
@@ -0,0 +1,80 @@
+.product-card {
+  border: 1px solid #fff;
+  width: 200px;
+  background-color: #fff;
```

Comment:
The background-color is set to #fff, which is white. Make sure this is the intended design choice, especially if the border is also white.